src/server/services/invoice/invoice.db.ts

```diff
@@ -7,14 +7,14 @@ import { Transaction } from 'sequelize/types';
 @singleton()
 export class InvoiceDB {
   async getAllInvoices(): Promise<InvoiceModel[]> {
-    return await InvoiceModel.findAll();
+    return await InvoiceModel.findAll({ order: [['status', 'DESC'], ['createdAt', 'DESC']]});
```

Comment:
What value does `status` have here again?

Reply:
It's to filter on the status which are "PAID", "PENDING", etc.